sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/samples/typescript/src/enableOpenTelemetryInstrumentation.ts

```diff
@@ -0,0 +1,53 @@
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { createAzureSdkInstrumentation } from "@azure/opentelemetry-instrumentation-azure-sdk";
+
+// Set-up and configure a Node Tracer Provider using OpenTelemetry
+import opentelemetry from "@opentelemetry/api";
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
+import { SimpleSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
+
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+registerInstrumentations({
+  instrumentations: [createAzureSdkInstrumentation()],
+});
+
+// Continue to import any Azure SDK client libraries after registering the instrumentation.
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyClient } from "@azure/keyvault-keys";
+
+// Load the .env file if it exists
+require("dotenv").config();
+
+async function main() {
+  const vaultName = process.env.AZURE_VAULT_NAME || "<VAULT-NAME>";
+  const url = `https://${vaultName}.vault.azure.net`;
+
+  const keyVaultKey = process.env.AZURE_VAULT_KEY_NAME || "MyKeyName";
+
+  const credential = new DefaultAzureCredential();
+  const keyClient = new KeyClient(url, credential);
+
+  // Tracing is now enabled using automatic span propagation with an active context.
+  await keyClient.getKey(keyVaultKey);
+
+  // If your scenario requires manual span propagation, all Azure client libraries
+  // support explicitly passing a parent context via an `options` parameter.
+  // Get a tracer from a registered provider, create a span, and get the current context.
+  const tracer = opentelemetry.trace.getTracer("my-tracer");
+  const span = tracer.startSpan("main");
+  const ctx = opentelemetry.trace.setSpan(opentelemetry.context.active(), span);
```

Comment:
same here and also `opentelemetry.context.active()` is likely a default value and there is overload without it. So probably it doesn't make sense to pass it